src/components/specific/models/models-manager/generic-models-manager/GenericModelsManager.vue

```diff
@@ -87,11 +91,15 @@ export default {
       validator: value => value.length > 0
     }
   },
-  setup(props) {
+  emits: ["edit", "get-current-tab"],
+  setup(props, { emit }) {
     const { downloadModels: download, updateModels } = useModels();
 
     const currentTab = ref({});
-    const selectTab = tab => (currentTab.value = tab);
+    const selectTab = tab => {
+      currentTab.value = tab;
+      emit("get-current-tab", tab);
```

Comment:
As we discussed the other day, event names should reflect the kind of action they are intended to trigger.
Here we provide the developer with the ability to **react** to a tab change, this imply that the tab has actually already been changed.
Thus I would rather use a name like `tab-changed` for this event.
The past form make it very clear that a changed occur and that we can trigger an action in reaction to that.
Conversely, a name `get-current-tab` suggest that an action should be performed to **get a tab**, which is not the case because tabs are managed internally by the component.
Let me know if you disagree or want to have further discussion about. :)